.github/workflows/main.yaml

```diff
@@ -349,3 +349,44 @@ jobs:
         run: |
           export DOCKER_BUILD_AND_SYSTEM_PRUNE="true"
           make image 
+  
+  container_runtime_test:
```

Comment:
It seems that there still some work need to do in this job. We should reference the steps of `e2e_test` job, rather than just installing the runtime.

Reply:
This is something left. I didn't get this.